.mvn/extensions.xml

```diff
@@ -7,8 +7,8 @@
       - local and remote cache
       build scans capture information see https://docs.gradle.com/enterprise/gradle-plugin/#captured_information
```

Comment:
```suggestion
      build scans capture information see https://docs.gradle.com/develocity/maven-extension/current/#capturing_goal_input_files
```

Reply:
I think you just want https://docs.gradle.com/develocity/maven-extension/#captured_information here. Updated in the latest push.

Reply:
indeed, thanks !